script.js

```diff
@@ -0,0 +1,36 @@
+const firstName = document.getElementById("firstName");
+const lastName = document.getElementById("lastName");
+const age = document.getElementById("age");
+const email = document.getElementById("email");
+const phoneNum = document.getElementById("phoneNum");
+const male = document.getElementById("male");
+const female = document.getElementById("female");
+let errorMsg = document.getElementsByTagName("p")[0];
```

Comment:
@Arzoo-25  Always try to give a `class` or `id` to the `tag` and get `document` by it in place of getting by `tag` for a best practice.

this comment is for the `<p>` tag